bridge/client/app/_components/ktb-create-secret-form/ktb-create-secret-form.component.spec.ts

```diff
@@ -18,10 +21,51 @@ describe('KtbCreateSecretFormComponent', () => {
 
     fixture = TestBed.createComponent(KtbCreateSecretFormComponent);
     component = fixture.componentInstance;
-    fixture.detectChanges();
+    dataService = fixture.debugElement.injector.get(DataService);
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should create secret with keptn-default scope', () => {
+    // given
+    const createButton = fixture.nativeElement.querySelector('[uitestid=keptn-secret-create-button]');
+    const spy = jest.spyOn(dataService, 'addSecret');
+
+    // when
+    component.getFormControl('name')?.setValue('test');
+    component.data?.controls[0].get('key')?.setValue('testKey');
+    component.data?.controls[0].get('value')?.setValue('testValue');
+    createButton.click();
+
+    // then
+    expect(spy).toHaveBeenCalled();
```

Comment:
I would change this to
```ts
const secret: Secret = new Secret();
secret.name = 'test';
secret.scope = component.scopes[1];
secret.data.push({key: 'testKey', value: 'testValue'});
expect(spy).toHaveBeenCalledWith(secret);
```
and remove the code below.

Maybe also make this secret global or create a mock-file because the values are re-used.